### DISCUSSION TITLE
Lucide-Icons out of date

### DISCUSSION THREAD

**Discussion by MichaelGorski** [tamagui/tamagui#2802] [General]
### Current Behavior

The @tamagui/lucide-icons is out of date and needs an update. Not all icons are exported and many are missing for example 

> ellipsis-vertical

### Expected Behavior

should have all icons and can be imported

### Tamagui Version

```markdown
1.100.0
```

### Platform (Web, iOS, Android)

```markdown
web, ios, android
```

### Reproduction

```markdown
try to import in your project EllipsisVertical

works from original import { EllipsisVertical } from 'lucide-react';

but not from Tamagui 
import { EllipsisVertical } from '@tamagui/lucide-icons'
```

**Comment by alexmandelshtam** [comment]
This seems like a good first issue, something I could look into as a new contributor?

**Comment by seanockert** [comment]
Ellipsis was the very first icon I tried to import. Frustrated that it's not included. Could you please support all commonly used Lucide Icons?

**Comment by natew** [comment]
Please someone do send a PR, not very hard to do.

**Reply by alexmandelshtam** [reply]
I'll be happy to. Using tamagui in a migration from native to RN/Expo right now and have wanted to get my feet wet with OSS. Will take a look at the contributing docs and give it a go.
